### DISCUSSION TITLE
Hello Dev

### DISCUSSION THREAD

**Discussion by Dev-SK01** [rudransh61/NextGenCSS-#20] [Q&A] [answered]
How to start ?

**Comment by rudransh61** [comment]
Thanks for joining us 
To start 

 - Read the code base 
 - code new feature you love
 - make a pull request

**Comment by rudransh61** [comment, marked as answer]
The code is in .src directory

It consists for many folders

The main css code for production is ./src/index.css

And all the sass files are included in 
./src/index.scss

The base directory includes some basic styles for web page .

The variables directory includes all the variables used in complete code base

(*You can add some comments and make code readable*) Issue


The css file converted from sass files is at .src/final/index.css/index.css
But this file is too large and not easy to load , 
So, we compressed our css file and pasted it into the main ./src/inde.css





SASS CODE 
THEN
.SRC/FINAL/INDEX.CSS/INDEX.CSS CODE GENERATED FROM SASS
THEN
.SRC/INDEX.CSS CODE GENERATED BY MINIFYING SRC/FINAL/INDEX.CSS/INDEX.CSS


You have to make changes on fork , write some sass and run 
./build.sh file to run all these changes .



Just write sass and run build.sh file it will automatically change it to minified css file


Thanks

**Comment by rudransh61** [comment]
Also star our repo

**Comment by rudransh61** [comment]
We will create a guide file related to structure of our code
